test/azure-pipelines.yml

```diff
@@ -77,11 +77,16 @@ jobs:
       curl -L https://raw.githubusercontent.com/InsightSoftwareConsortium/ITK/dashboard/itk_common.cmake -O
     displayName: 'Fetch CTest driver script'
 
+  - bash: |
```

Comment:
cookiecutter should not be run in CI, but just once and customized template used to update PhaseSymmetry.